src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -95,13 +106,35 @@
 			</NcActionButton>
 		</template>
 
-		<!-- confirmation required to delete conversation -->
-		<template v-if="isDialogOpen" #extra>
-			<NcDialog :open.sync="isDialogOpen"
+		<!-- confirmation required to leave / delete conversation -->
+		<template v-if="isLeaveDialogOpen || isDeleteDialogOpen" #extra>
+			<NcDialog v-if="isLeaveDialogOpen"
+				:open.sync="isLeaveDialogOpen"
+				:name="t('spreed', 'Leave conversation')">
+				<template #default>
+					<p>{{ dialogLeaveMessage }}</p>
+					<p v-if="supportsArchive && !item.isArchived">
+						{{ t('spreed', 'You can archive this conversation instead.') }}
+					</p>
+				</template>
+				<template #actions>
+					<NcButton v-if="supportsArchive && !item.isArchived" type="secondary" @click="toggleArchiveConversation">
+						{{ t('spreed', 'Archive instead') }}
```

Comment:
```suggestion
						{{ t('spreed', 'Archive conversation') }}
```

Better and aligning with `DangerZone`